src/pages/home/home.scss

```diff
@@ -0,0 +1,74 @@
+@import "~theme/base/variables";
+
+.homeContainer {
+  height: 100%;
+}
+
+.navigationBar {
+  padding: 1.25em 0em;
+}
+
+.navigationBar .leftNavLinks {
+  margin-left: 4em;
+}
+
+.navigationBar .navLinks {
+  display: inline-block;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #179A6F;
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  font-size: 1rem;
+  margin-right: 2em;
+}
+
+.mainHeaderBackground {
+  background: url('../../assets/main-header-background.png') no-repeat;
```

Comment:
I'm not sure if the extra `../../` are needed. I think webpack will resolve the image without those, please try that and let us know if that's the case.

Reply:
Resolved.